ui/projects/DeployRepository.qml

```diff
@@ -0,0 +1,48 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+
+Item {
+    id: root
+
+    property var repository;
+    property var envNameToDeploy;
+
+    height: Math.max(contents.height)
+    width: Math.max(contents.width)
+
+    anchors.horizontalCenter: parent.horizontalCenter
+
+    Column {
```

Comment:
Please consider to use ColumnLayout in a next release

Reply:
Ok, I never used ColumnLayout, I will try to use it for my next release.